poc_v2.1_revised/app.py

```
@@ -0,0 +1,82 @@
+            fn=lambda q, r: ask_star(q.replace("💬 ", ""), r),
+            inputs=[followup_btn1, role_input],
+            outputs=[output, followup_heading, followup_btn1, followup_btn2]
+        )
+
+        followup_btn2.click(
+            fn=lambda q, r: ask_star(q.replace("💬 ", ""), r),
+            inputs=[followup_btn2, role_input],
+            outputs=[output, followup_heading, followup_btn1, followup_btn2]
+        )
+
+    iface.launch(server_name="0.0.0.0", server_port=7860,share=True)
```

Ruff (0.15.7)

[error] Possible binding to all interfaces

(S104)